src/PresentationalComponents/ZeroState/AppZeroState.test.js

```diff
@@ -98,7 +98,7 @@ describe('AppZeroState component', () => {
 
     it('DOES render zero state when there are no systems in default request', async () => {
         const mockedGet = jest.spyOn(useAxiosWithPlatformInterceptors(), 'get')
-        .mockResolvedValue({ data: { total: 0 } });
+        .mockResolvedValue({ total: 0 });
```

Comment:
Has the backend response changed? normally the response does have a `data` prop and a `meta` prop, or?

Reply:
I kept up with BE team and it seems that the recent refactor may have just exposed the issue. It still raises some questions as we had this in there for quite some time with no issues, but so far it doesnt look like anything on the BE has changed.

https://redhat-internal.slack.com/archives/CQFKM031T/p1725391402133989

The sentry error started the day we released which further leads me to think we just exposed an pre-existing issue, though its still odd that the edge case wasnt hit.

Reply:
@bastilian pinging this up again